CHANGES.md

```diff
@@ -132,20 +130,17 @@ Added test case.
 - Source compiles under JDK 1.5
 
 - Better defined classloading precedence:
-
-    in-script evaluated class (scripted class)
-    in-script added / modified classpath
-
-    optionally, external classloader
-    optionally, thread context classloader
-
-    plain Class.forName()
-    source class (.java file in classpath)
+    * in-script evaluated class (scripted class)
+    * in-script added / modified classpath
+    * optionally, external classloader
+    * optionally, thread context classloader
+    * plain `Class.forName()`
+    * source class (.java file in classpath)
```

Comment:
Indenting lists is not really necessary. Whether they are easier to read like this is open to discussion.

Reply:
I interpreted the original text as a list within a parent list element. Indentation is required to render it that way in markdown.
Here's what I think we want:
![image](https://user-images.githubusercontent.com/66449124/191612077-5c3d60c7-263d-42a7-803c-def2b3c81ad9.png)

Here's how it looks with indentation removed, which has a very different meaning:

![image](https://user-images.githubusercontent.com/66449124/191612373-4ed59b25-7486-493c-8841-1d056386fd55.png)

Reply:
Good point. Somehow I missed this.